apps/prairielearn/elements/pl-drawing/pl-drawing.mustache

```diff
@@ -12,7 +12,7 @@
 {{^parse_error}}
 <div id="pl-drawing-{{uuid}}" class="pl-drawing-container pl-requires-svg-mathjax">
     <div>
-        <canvas width="{{width}}" height="{{height}}"> </canvas>
+        <canvas width="{{width}}" height="{{height}} aria-label="Drawing container"> </canvas>
```

Comment:
What's the benefit of adding a label to the canvas? This isn't an input and isn't focusable.

Reply:
Removed in [6f54747](https://github.com/PrairieLearn/PrairieLearn/pull/10514/commits/6f547473f9ac91f949f5e3f77ced434e15291109).